tracer/src/Datadog.Trace/Ci/TestModule.cs

```diff
@@ -605,4 +635,75 @@ internal void RemoveSuite(string name)
             _suites.Remove(name);
         }
     }
+
+    internal bool EnableIpcClient()
+    {
+        if (_fakeSession != null || Tags.SessionId == 0)
+        {
+            return false;
+        }
+
+        try
+        {
+            var name = $"session_{Tags.SessionId}";
```

Comment:
Random thought, we're never going to get anywhere near file path length issues are we? Should we add guards for that at least in the `IpcClient`? 🤔

Reply:
I don't think we are near to that, sessionId is a normal 64bit SpanId number.
But I'll add that to the things TODO in the upcoming PR.